src/emulator/gui/src/gui.cpp

```diff
@@ -153,11 +153,12 @@ static void init_background(State &gui, const std::string &image_path) {
     stbi_uc *data = stbi_load(image_path.c_str(), &width, &height, nullptr, STBI_rgb_alpha);
 
     if (!data) {
-        LOG_INFO("Could not load image from {}.", image_path);
+        LOG_ERROR("Invalide format image from {}, check your image format.", image_path);
```

Comment:
What's with this weird French append

```suggestion
        LOG_ERROR("Invalid image format or corrupted image: {}.", image_path);
```